src/extensions/mountainview/MVSortingView/CurationControl.tsx

```diff
@@ -59,18 +74,28 @@ const CurationControl: FunctionComponent<Props & SizeMeProps> = ({ selection, se
     const labels = Object.keys(labelCounts).sort()
     const labelRecords: LabelRecord[] = labels.map(label => ({
         label,
-        partial: labelCounts[label] < selection.selectedUnitIds.length ? true : false
+        partial: labelCounts[label] < selectedUnitIds.length ? true : false
     }))
     const paperStyle: React.CSSProperties = {
         marginTop: 25,
         marginBottom: 25,
         backgroundColor: '#f9f9ff'
     }
-    const enableApply = selection.selectedUnitIds.length > 0
+    const unitsAreInMergeGroups = (unitIds: number[]) => {
```

Comment:
Returns true if all the units are in some merge groups. We only want to give the "unmerge" option if this is the case.